MAUI/ImageEditor/Effects.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: Image Effects in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about Image Effect support in Syncfusion .NET MAUI Image Editor (SfImageEditor) control.
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+# Image Effect in .NET MAUI Image Editor (SfImageEditor)
+
+Using the image editor control, you can add effects such as Hue, Saturation, Brightness, Contrast, Blur, and Sharpen to the image. These effects can be applied from toolbar or using the ImageEffect method. The [`ApplyImageEffect`] method contains two arguments: [`ImageEffect`] and [`EffectValue`]. The ImageEffect is an Enum, which contains the following effects:
+
+* Brightness
+* Blur
+* Contrast
+* Exposure
+* Hue
+* Saturation
+* Sharpen 
+* Opacity
+* None
+
+The EffectValue are the corresponding ImageEffect values, which varies for each effect, and they are explained as follows.
+
+N> The Image Effect enum also contains “None” option, which removes all the previously applied effects, which are not saved.
+The ImageEffect method only applies the effect to preview image, if you want to save the applied effect you should call SaveEdits method.
+
+## Brightness
+
+Brightness is used to adjusts the overall lightness or darkness of the image. The value of brightness effect ranges from -1 to 1 and the default value is 0.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect.Brightness, -0.6);
+    . . .
+}
+
+{% endhighlight %}
+
+
+## Blur
+
+Blur is used to Creates a soft and unfocused appearance by reducing the image's sharpness. The value of blur effect ranges from 0 to 1 and the default value is 0.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect.Blur, 0.5);
+    . . .
+}
+
+{% endhighlight %}
+
+
+## Contrast
+
+Contrast is used to increases or decreases the difference between light and dark areas, making the image more visually distinct. The value of contrast effect ranges from -1 to 1 and the default value is 0.
+
+{% highlight C# %}
+
+public MainPage()
+{               
+    . . .
+    this.imageEditor.ImageEffect(ImageEffect. Contrast, -0.8);
+    . . .
+}
+
+{% endhighlight %}
+
+
+## Exposure
+
+Exposure is used to alters the overall brightness and darkness levels of the image. The value of the exposure effect ranges from -1 to 1 and the default value is 0.

```

Comment:
Exposure is used to alter the overall brightness and darkness levels of the image. The value of the exposure effect ranges from -1 to 1 and the default value is 0.

Reply:
updated